src/backend/public/index.js

```diff
@@ -512,31 +515,34 @@ function accumulatePixels() {
         startTime = new Date();
         numberOfAccumulatedPixels.addPixels(1);
         updateAccPixels();
+        startCountdown();
     }, cooldownTime); 
 }
 
 function startCountdown() {
-    // NOTE: cooldownTime is in MILLISECONDS 
-    let elapsed = new Date() - startTime; // in ms
-    let frac = 1 - ((elapsed % cooldownTime) / cooldownTime); // fraction of cooldownTime left till next accumulated pixels + 1
-    let secs = Math.floor(frac * cooldownTime/1000); 
-    console.log("Elapsed timing: " + elapsed + "\n Countdown starting with ", frac, secs);
-    let mins = Math.floor(secs / 60);
-    let secsDisp = secs % 60;
-    // setInterval takes one second to start 
-    countdownMin.innerText = mins;
-    countdownSec.innerText = secsDisp; 
-    let interval = setInterval(function() {
-        if (secs === 0) {
-            clearInterval(interval);
-        } else {
-            secs--;
-            mins = Math.floor(secs / 60);
-            secsDisp = secs % 60;
-            countdownMin.innerText = mins;
-            countdownSec.innerText = secsDisp;
-        }
-    }, 1000);
+    if (numberOfAccumulatedPixels.getPixels() < 10) {
```

Comment:
Number '10' should be abstracted out to a MAX_ACCUMULATED_PIXELS value in accumulatedPixels.js - also, make sure that all occurrences of '10' in your entire index.js are abstracted out.